### PR TITLE
KFSPTS-6210: fix error with unit tests running in local dev and ignor…

### PR DESCRIPTION
**Pull request by bhutchinson**
…e test/ directory created by unit tests

### CODE REVIEW

File: src/test/java/edu/cornell/kfs/fp/document/CuDisbursementVoucherDocumentTest.java [hunk 1]
```diff
@@ -120,6 +121,11 @@ public static void setUp() throws Exception {
         cuDisbursementVoucherDocument.setDocumentHelperService(documentHelperService);
     }
 
+    @AfterClass
+    public static void tearDown() {
+        cuDisbursementVoucherDocument.setDocumentHelperService(null);
```
Comment (db483) [inline src/test/java/edu/cornell/kfs/fp/document/CuDisbursementVoucherDocumentTest.java]: Just trying to understand this fix.  Why was the documentHelperService the only service that had to be reset to null?

Comment (bhutchinson) [inline src/test/java/edu/cornell/kfs/fp/document/CuDisbursementVoucherDocumentTest.java]: Because it was the only one that was mocked using EasyMock.

Comment (db483) [inline src/test/java/edu/cornell/kfs/fp/document/CuDisbursementVoucherDocumentTest.java]: Thank you.

**Comment by db483** [comment]
Code changes look good and all unit tests passed. Will merge it.

**Pull request merged**